docs/JENKINS.adoc

```diff
@@ -278,25 +249,14 @@ when you want to do some custom changes.
 
 ==== Deploying infra jars to a different location
 
-It's enough to provide the `distribution.management.release.url` system prop to Maven.
-Example for deploying to Artifactory at IP `192.168.99.100`
-
-Github Eureka:
+It's enough to set the `ARTIFACTORY_URL` environmental variable before
+executing `tools/deploy-infra.sh`. Example for deploying to Artifactory at IP `192.168.99.100`
 
 [source,bash]
 ----
-git clone https://github.com/spring-cloud-samples/github-eureka
-cd github-eureka
-./mvnw clean deploy -Ddistribution.management.release.url=http://192.168.99.100:8081/artifactory/libs-release-local
-----
-
-Github Stub Runner:
-
-[source,bash]
-----
-git clone https://github.com/spring-cloud-samples/github-analytics-stub-runner-boot
-cd github-analytics-stub-runner-boot
-./mvnw clean deploy -Ddistribution.management.release.url=http://192.168.99.100:8081/artifactory/libs-release-local
+git clone https://github.com/spring-cloud/spring-cloud-pipelines
+cd spring-cloud-pipelines/
+ARTIFACTORY_URL="http://192.168.99.100:8081/artifactory/libs-release-local" ./tools/deploy-infra.sh
```

Comment:
It's already written :) This example is for "Deploying infra jars to a different location" chapter.